elements/urth-core-behaviors/jupyter-widget-behavior.html

```diff
@@ -54,7 +54,7 @@
                 ).then(function(model) {
                         //This check is to protect against a timing in ipywidgets where the comm can be close
                         //by the time it gets to resolve this promise
-                        if(!model.comm.kernel.comm_manager.comms[model.comm.comm_id]){
+                        if(model.comm && !model.comm.kernel.comm_manager.comms[model.comm.comm_id]){
```

Comment:
now... I would think that the model will always have a comm. Don't mind checking, but why would it be undefined?